### DISCUSSION TITLE
Support `top-[calc(80px+8px)]`

### DISCUSSION THREAD

**Discussion by sondh0127** [unocss/unocss#120] [Q&A]
Does uno-preset support `top-[calc(80px+8px)]` ?

**Comment by sondh0127** [comment]
![image](https://user-images.githubusercontent.com/62163604/142354265-c12bb3a8-b7cc-4770-b88f-47f3307caf89.png)
Currently I use twind (shim ver), which support this case well.
Is that posible to add this feature to uno.

Currently, uno support runtime version. Can we use runtime uno and regular-uno together ?

**Reply by hannoeru** [reply]
You will need to add all passable class to safelist.

<img width="206" alt="スクリーンショット 2021-11-20 14 48 11" src="https://user-images.githubusercontent.com/16288797/142716007-d20e8705-1005-463c-b91f-2e5d17e59fac.png">

> Can we use runtime uno and regular-uno together ?

Why? not recommend.

**Comment by hannoeru** [comment]
Yes we do.

=> [playground](https://unocss.antfu.me/?html=DwEwlgbgBAxgNgQwM5ILwCIAuB7ADgWgG0YE4YAKADgAZcAPAakvoEoBddAPgCgooBXAHbYYKbsAD04CJyA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHCjjUrAFxwA2itV7azeqxjUAtOrGpaAGkRxmEMdB20oqZLTIBdA-6kAJRAA&options=N4IgzgLgTglgxhEAuaBXApgXyA)
